tests/test_multi_resources.py

```diff
@@ -0,0 +1,39 @@
+import sky
```

Comment:
Should we have it in the optimizer test instead? Also, do we want to add it to CI?

Reply:
moved.